pmd-apex/src/main/resources/rulesets/apex/ruleset.xml

```diff
@@ -2,443 +2,50 @@
 <ruleset xmlns="http://pmd.sourceforge.net/ruleset/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Default ruleset used by the CodeClimate Engine for Salesforce.com Apex" xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0 https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
    <description>Default ruleset used by the Code Climate Engine for Salesforce.com Apex</description>
 
-   <!-- COMPLEXITY -->
```

Comment:
I wouldn't have expected any change in this file... what is changed here?

Reply:
this file was shown in merge conflict, so I pulled this from the upstream master.

Reply:
This now adds an unrelated change into this PR... which is not desired.

Solving merge conflict can be done by either merging in the master branch into the PR-branch or rebasing the PR-branch against master. In both cases, you deal with the merge conflict, when you need to (at the time of merging/rebasing).

I'll undo the changes in this file before I merge this PR (since this PR should not change at all this file and thus completely avoids any merge conflicts in this file....).